test/CppTests/Tests/BreakpointTests.cs

```diff
@@ -137,7 +137,7 @@ public void FunctionBreakpointsBasic(ITestSettings settings)
                               .AfterContinue();
 
                 this.Comment("Remove and replace third initial function breakpoint while in break mode");
-                functionBreakpoints.Remove("a()");
+                functionBreakpoints.Remove("a");
                 functionBreakpoints.Add("b()");
```

Comment:
Did you remove the `()` because your version of GDB didn't like them? If so, do you want to remove it from `b()` as well?

Reply:
Oh, I see. Nevermind.